postgres/rdspostgres/rdspostgres_test.go

```diff
@@ -60,6 +60,27 @@ func TestOpen(t *testing.T) {
 	if err := db.Close(); err != nil {
 		t.Error("Close:", err)
 	}
+
+	// Temporary test part for temporary OpenWithUrl method
```

Comment:
So far, we've largely been leaving `URLOpener` types mostly untested because the logic is usually trivial enough to be reviewable, the cost for test in integration is high, and then the contortions to the API required to test the logic in isolation make the control flow harder to read.

Since the logic required here is mostly matching the parts of a `*URL` to the appropriate slots in the input to `rdspostgres.Open`, I think we should drop the tests here.